components/context/context.css

```diff
@@ -0,0 +1,6 @@
+.context {
+    min-height: 1036px;
+    background: #F9FAFC;
```

Comment:
I was thinking about moving the color in main color dictionary. 
<img width="243" alt="Screenshot 2023-04-17 at 21 07 03" src="https://user-images.githubusercontent.com/96296081/232572751-1f8069d7-4c9d-4e88-9068-d8bdd1b852a0.png">

Should it have beed done?

Reply:
yes, but not in this PR, later.